README.md

```diff
@@ -1,168 +1,123 @@
 # 3D Toolkit
 
-A toolkit for building powerful stereoscopic 3d experiences that run on the cloud and stream to devices. :muscle: :eye: :cloud:
+A toolkit for creating 3D experiences that are traditionally out of reach on low-powered devices. :muscle: :eye: :cloud:
```

Comment:
did i already express my concern about removing cloud wording from this? I think it could be misleading because there any many other ways to improve perf of 3d on low-powered devices (smart poly reduction, better engine, etc)